src/App.js

```diff
@@ -35,19 +37,23 @@ export default function App() {
     setResumedState(_resumedState);
   }, []);
 
-  useEffect(() => {
-    resumeState();
+  useEffect(() => { 
+      resumeState();
   }, [resumeState]);
```

Comment:
Originally not including dependencies ensured that it only ran once on load. Does the new Permalinks behavior require this in dependencies to function?

Reply:
> Originally not including dependencies ensured that it only ran once on load. Does the new Permalinks behavior require this in dependencies to function?

This specific dependency was not added by me. But I did add a new state (filteredState) that comes from usePermalinkState which consumes the resumedState. That is for making the permalinks load data faster if it matches what is stored on local.

So what usePermalinkState does is:
- If there is local data and the permalink matches what is in local, it takes the data from local.
- If there is local data and the permalink doesn't match what is in local, it fetches data from DCS.
- If there is no local data and there is a permalink, it fetches data from DCS.
- If there is no local data and there is no permalink, it will return the same resumeState returns when there is no data in local.

Reply:
I guess we could merge permalinkState and resumedState so it only renders once.